build_emacs_ng.sh

```diff
@@ -1,8 +1,12 @@
 sudo add-apt-repository -y ppa:ubuntu-toolchain-r/ppa
+sudo apt update
 sudo apt install -y autoconf make checkinstall texinfo libxpm-dev libjpeg-dev \
      libgtk-3-dev libgif-dev libtiff-dev libpng-dev libgnutls28-dev libncurses5-dev \
-     libsystemd-dev libjansson-dev libharfbuzz-dev libgccjit-10-dev gcc-10 g++-10
-sudo apt update
+     libsystemd-dev libjansson-dev libharfbuzz-dev libgccjit-10-dev gcc-10 g++-10 libxt-dev \
+     libclang-10-dev 
+
+sudo snap install rustup --classic
```

Comment:
The release workflow has already install rust via Gihub action.
https://github.com/emacs-ng/emacs-ng/blob/415de78e6d4fdd2c7f3f28cafe37ac1be710ee41/.github/workflows/release.yml#L20

Can you remove that step there?

Also, `snap` is fairly new installation method and not available on all distro. Can you use something like instead?
```suggestion
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
```

Reply:
Thanks, I only tested snap on ubuntu.